<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>991</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>991</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Arabic Naskh</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="text">
     <string>THERMOFORMING MACHINE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Arabic Naskh</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="text">
     <string>TEMPERATURE CONTROL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Arabic Naskh</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_on">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Desktop/Botones/On.png</normaloff>
      <disabledoff>../Desktop/Botones/OnA.png</disabledoff>
      <disabledon>../Desktop/Botones/OnA.png</disabledon>
      <activeoff>../Desktop/Botones/On.png</activeoff>
      <activeon>../Desktop/Botones/On.png</activeon>../Desktop/Botones/On.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vacuum">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Desktop/Botones/Vacuum.png</normaloff>
      <normalon>../Desktop/Botones/VacuumA.png</normalon>../Desktop/Botones/Vacuum.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_preHeat">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Desktop/Botones/Preheat.png</normaloff>
      <normalon>../Desktop/Botones/Preheat.png</normalon>
      <disabledoff>../Desktop/Botones/PreheatA2.png</disabledoff>
      <disabledon>../Desktop/Botones/PreheatA2.png</disabledon>../Desktop/Botones/Preheat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Desktop/Botones/StopA.png</normaloff>../Desktop/Botones/StopA.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Arabic Naskh</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>21</width>
      <height>151</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_resistance_temp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Temperature &lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set Point &lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdnumber_SetPoint_temp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Arabic Naskh</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 59, 180);</string>
    </property>
    <property name="text">
     <string>VACUUM CONTROL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_termopar1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_termopar_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_termopar_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Termocouples&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox_auto_material">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Acrilico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PET</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Termoplastico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="spinBox_manual_temp">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>280</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Material</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Semi-Automatic</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>pushButton_on</zorder>
   <zorder>pushButton_vacuum</zorder>
   <zorder>pushButton_preHeat</zorder>
   <zorder>pushButton_Stop</zorder>
   <zorder>label_4</zorder>
   <zorder>line_3</zorder>
   <zorder>progressBar</zorder>
   <zorder>lcd_resistance_temp</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>lcdNumber_2</zorder>
   <zorder>label_7</zorder>
   <zorder>lcdnumber_SetPoint_temp</zorder>
   <zorder>line_5</zorder>
   <zorder>label_10</zorder>
   <zorder>radio_termopar1</zorder>
   <zorder>radio_termopar_2</zorder>
   <zorder>radio_termopar_3</zorder>
   <zorder>label_11</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
